make/verify-dependencies.mk

```diff
@@ -0,0 +1,12 @@
+.PHONY: verify-dependencies
+## Runs commands to verify after the updated dependecies of toolchain-common/API(go mod replace), if the repo needs any changes to be made
+verify-dependencies: tidy vet test lint-go-code
```

Comment:
you can also run 
```
go test ./... -skip '.*'
```
to verify that all tests can be compiled

Reply:
Thanks, This is a good add-on , 

added it